apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/commons/AbstractSamhsaMatcherTest.java

```diff
@@ -500,7 +531,7 @@ public static Stream<Arguments> samhsaCodingTest() {
             true,
             false,
             true,
-            "ICD9 System Coding not correctly marked samhsa."),
+            "ICD9 System Coding incorrectly marked samhsa."),
```

Comment:
I don't think we want this change but the existing language is confusing so making some type of change is probably warranted. The test is simulating a CodeableConcept with an ICD 9 SAMHSA code and confirming that the CodeableConcept as a whole is considered to be a SAMHSA coding. This text is the error message that will be returned if that is *not* the case. So the coding should be marked SAMHSA but wasn't. Perhaps the message on these scenarios should be "ICD 9 System Coding expected to be marked samhsa but wasn't."

Reply:
Updated for all cases